### DISCUSSION TITLE
Horipad Xbox One Controller

### DISCUSSION THREAD

**Discussion by MichiMaster1** [cathery/sys-con#182] [Controller requests]
It doesn't recognize the controller. Syscon is correctly installed.

Atmosphere 0.10.5
Firmware 9.2.0

**Comment by RigleGit** [comment]
Maybe you can use the tool to give him the PID and VID so it's easier for him to add them!

Steps:
1.     Download this diagnostics applet https://files.catbox.moe/re60qo.rar
2.     Run it from the homebrew menu.
3.     Plug in the controller into the switch.
4.     Press Y to scan and then take a screenshot.
5.     Post the screenshot here and erase these steps.
